### PR TITLE
Version fix

### PR DESCRIPTION
**Pull request by mcanoy**
same as git-api update
* fix versioning
* comes from env var which matches the tag
* if the tag is lastest or master append the commit.
* update release tag var

### CODE REVIEW

File: src/main/java/com/redhat/labs/omp/service/VersionService.java [hunk 1]
```diff
@@ -40,8 +40,8 @@ public Version getGitApiVersion() {
         }
         version.setApplication("omp-git-api-container");
         
-        if(version.getGitTag().startsWith("v")) {
-            version.setVersion(version.getGitTag());
+        if(version.getGitTag().equals("master") || version.getGitTag().equals("latest")) {
+            version.setVersion(version.getGitTag() + "-" + version.getGitTag());
```
Comment (dwasinge) [inline src/main/java/com/redhat/labs/omp/service/VersionService.java:44]: Just clarification.  Is this supposed to be version.getGitTag() + version.getGitCommit()?  I may just not have the context right here.

Comment (jacobsee) [reply src/main/java/com/redhat/labs/omp/service/VersionService.java:44]: Definitely missed that... good catch

Comment (mcanoy) [reply src/main/java/com/redhat/labs/omp/service/VersionService.java:44]: yeah. basically - if you use master or latest then you are not on a release. So it'll just say

master-asdf9a8dfk3 in that case. It replaces checking if v*. Essentially means this is deployed in dev or test.

Comment (dwasinge) [reply src/main/java/com/redhat/labs/omp/service/VersionService.java:44]: 👍

**Pull request merged**